gomint-server/src/main/java/io/gomint/server/command/internal/TitleCommand.java

```diff
@@ -64,17 +65,16 @@ public CommandOutput execute( CommandSender player, String alias, Map<String, Ob
         if( arguments.containsKey( "type" ) ) {
             switch( (String) arguments.get( "type" ) ) {
                 case "title":
-                    target.sendTitle( (String) arguments.get( "titleText" ) );
+                    target.sendTitle( (String) arguments.get( "text" ) );
                     break;
                 case "subtitle":
-                    target.sendTitle( null, (String) arguments.get( "text" ) );
+                    target.sendTitle( "", (String) arguments.get( "text" ) );
```

Comment:
don't you overwrite the title now?

Reply:
@Janmm14 https://github.com/GoMint/GoMint/pull/301/files#diff-930bb11bbb10af48ae040e72a977a72eR1506

Reply:
@Janmm14 https://github.com/lukeeey/GoMint/blob/424fd579d31fb8f435122e1d614c71f427b42f92/gomint-server/src/main/java/io/gomint/server/entity/EntityPlayer.java#L1506